bsp4j/src/main/java/ch/epfl/scala/bsp4j/Protocol.xtend

```diff
@@ -32,11 +32,13 @@ class BuildTargetCapabilities {
   @NonNull Boolean canCompile
   @NonNull Boolean canTest
   @NonNull Boolean canRun
+  @NonNull Boolean canDebug
```

Comment:
making this `@NonNull` will make this change non-backward compatible

Reply:
removed this and regenerated `xtend`

Reply:
Thanks! I think in the Scala version it would need to be `Option[Boolean]` also, even though that is kind of ugly ...

Reply:
I'm not super familiar with the syntax, but is there a way to make it a default value false?

Reply:
you could do `canDebug: Boolean = false` but I'm not sure if/how that interacts with the serialization? I guess if that passes tests it's fine ;)